### ISSUE TITLE
Speed up fake subs response in demo

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1303]
- [x] response takes a long time, fix that
- [x] also forward emails from the account it's linked to to Joe so he can fiddle with it

**Comment by JosephMcArthur** [comment]
Once this is done, it's on @me to

- [ ] change demo account to form link

**Comment by markmacgillivray** [comment]
I have changed the code to be fast. But please have a test and see if the demo still does all the things you expect. Also, not sure what you mean about wanting emails forwarded to you. What emails do you mean? The demo does not actually go through the usual whole process now, so I am not sure it will send any emails that you would want to see.

**Comment by JosephMcArthur** [comment]
Thanks Mark! The email stuff was about setting up the forwarder so I can loggin. For some reason you did that in another issue but it's all the same to me :) 

It does what I expect!
